README.md

```diff
@@ -1 +1,55 @@
-# momento-php-redis-client
\ No newline at end of file
+# Welcome to Momento php-redis drop-in replacement
+
+Welcome to the Momento PHP-Redis Drop-in Replacement package! This package is a wrapper around the PHP-Redis extension that allows integration with Momento's caching service. You can use this as a direct replacement for PHP-Redis, while leveraging the benefits of Momento's serverless cache.
+
+## Pre-requisites
+
+Before running the integration tests, ensure that the Momento API Key is available in your environment. If you don't have one, you can get it from the  [Momento Console](https://console.gomomento.com).
+Additionally, create a .env file at the root of the project with the following contents:
+
+```bash
+TEST_REDIS=true # false if you would like to use momento
+REDIS_HOST="127.0.0.1" # Redis host
+REDIS_PORT=6379 # Redis port
+TEST_CACHE_NAME="test-cache" # Cache name
+```
+
+## Installation
+
+To set up the integration tests, you'll need two Docker containers: one for running Redis and the other for running the integration tests.
```

Comment:
To keep it simple we can just run Redis in a docker container and run the integration tests directly, right? Or is there something about the php build that makes it better to run in docker?

Reply:
yes, installing php on mac is a hassle. And therefore, we have always used docker for php development and testing.